### DISCUSSION TITLE
Componente Payment styling

### DISCUSSION THREAD

**Discussion by renerlemes** [mercadopago/sdk-react#101] [General]
Existe alguma maneira de alterar o style dos input como o número do cartão, validade...? Não encontrei a propriedade nas customizações quando inicio o componente.
Estou querendo alterar o font-family e apenas nos campos de número do cartão, validade e código de segurança que não dá certo, nos outros, o formulário obteve automático do meu css.

**Comment by AleFossati** [comment]
Olá @renerlemes. Por padrão, o CardPayment e o Payment Brick utilizam o `font-family` que está nos seu container, porém a fonte já deve estar no elemento **ANTES** de carregar o brick.

Isso é necessário porque por questões de segurança, os campos do cartão (número, validade e cvv) ficam dentro de um `iframe` e as estilizações não propagam para eles. A customização da font só funciona no render do brick, pois na criação do iframe em tela o brick propaga alguns atributos CSS pra dentro dele.

Resumindo, qualquer mudança de CSS depois que o brick já estiver renderizado em tela, não vai funcionar para os campos de cartão.

Espero que isso te ajude a resolver este inconveniente. Mas caso ainda tenha dúvidas, não hesite em nos perguntar novamente =)

**Reply by renerlemes** [reply]
@AleFossati entendi tudo e obrigado pela excelente explicação e rápida resposta.
